test-suite-data.json

```diff
@@ -252,7 +252,7 @@
     "is_invalid": false
   },
   {
-    "description": "slash /// after type  is not significant",
+    "description": "slash /// after scheme is not significant",
```

Comment:
Thanks for the correction. This is a carryover from a time before PackageURLs began with pkg:/